sys/phydat/phydat_str.c

```diff
@@ -64,6 +64,8 @@ const char *phydat_unit_to_str(uint8_t unit)
         case UNIT_TEMP_K:   return "K";
         case UNIT_LUX:      return "lx";
         case UNIT_M:        return "m";
+        case UNIT_M2:       return "sqm";
+        case UNIT_M3:       return "cbm";
```

Comment:
I know the maths, but that is a problem of scale vs unit dimension, not of the string conversion.

So we either have to fix that, or can't use UNIT_M2 and UNIT_M3